packages/patterns/.cfignore

```diff
@@ -0,0 +1,4 @@
+*
+!manifest.yml
```

Comment:
Doesn't cloud foundry need a `manifest.yml` to work? Or does the CLI take care of the `manifest.yml` before uploading to cloud foundry?

Reply:
I think this is actually carry over from the carbon repo. we do need a manifest file for cloud foundry, but since we are transitioning to EI we can clean this up later.